lib/prawn/templates.rb

```diff
@@ -246,4 +258,4 @@ def load_object_graph(hash, object)
 Prawn::Document::VALID_OPTIONS << :template
 Prawn::Document.extensions << Prawn::Templates
 
-PDF::Core::ObjectStore.send(:include, Prawn::Templates::ObjectStoreExtensions)
```

Comment:
I  see, actually I previously had support for ruby 2.0.0+ only. Should I try to maintain 1.9.3 until praw drops it?